libmscore/select.h

```diff
@@ -42,6 +42,7 @@ struct ElementPattern {
       int voice;
       const System* system;
       bool subtypeValid;
+      TDuration duration = TDuration();
```

Comment:
In C++ you don't need to set default value explicitly in this case, because for user data structures like classes default values are constructed by default constructor already.